### DISCUSSION TITLE
Automatisches WLAN Reconnect geht nicht

### DISCUSSION THREAD

**Discussion by ms1958** [tbnobody/OpenDTU#2305] [General]
Guten Morgen, ich habe das Problem, wenn die DTU das WLAN Signal zu meinem netzwerk verloren hat (z.B. Stromausfall oder reboot), das sich die DTU (Akkugepuffert)  nicht neu verbindet. Ich muss sie dann immer kurz stromlos machen. Das ist nervig, besonders, wenn man gar nicht zu Hause ist. Firmware V24.8.5 hab ich auf der DTU. - Schön wäre ein Parametrierbarer Reconnect mit einem Bekannten WLAN, wenn der AP der DTU aktiv ist.

**Comment by borland123** [comment]
Moin, ich habe das gleiche Problem.
Meine DTU verliert ab und zu (so alle 2 Wochen) die Verbindung zum WLAN und verbindet sich dann nicht mehr neu, 
Ich muss dann auch den Stecker der DTU ziehen. 
Die Verbindung zu den Wechselrichter bleibt aber immer bestehen das sehe ich am Display der DTU.
Neuste FW ist drauf, andere Geräte die sich in der Náhe der DTU befinden und im WLAN hängen habe das Problem nicht. 

Kann jemand helfen oder soll ich noch etwas liefern?
